### PR TITLE
#65 update dashboard functionality

### PR DESCRIPTION
**Pull request by yastapenko-ScnSoft**
#65

### CODE REVIEW

**Comment by yastapenko-ScnSoft** [comment]
Done as described in last commit #65.

**Comment by mturalenka** [comment]
@yastapenko-ScnSoft please resolve phpcs errors first

File: ext_tables.sql [hunk 1]
```diff
@@ -22,11 +22,9 @@ CREATE TABLE tt_laterpay_payment_history (
 	UNIQUE KEY  (mode, hash),
 ) ENGINE=InnoDB;
 
-CREATE TABLE tt_laterpay_post_views (
-	post_id           INT(11)         NOT NULL,
+CREATE TABLE tt_laterpay_content_views (
+	content_id           INT(11)         NOT NULL,
```
Comment (mturalenka) [inline ext_tables.sql:26]: fields are not aligned

**Comment by mturalenka** [comment]
Notice: Undefined index: comment_opener in /home/travis/.phpenv/versions/5.5.24/pear/PHP/CodeSniffer/Standards/TYPO3SniffPool/Sniffs/Commenting/NoAuthorAnnotationInFunctionDocCommentSniff.php on line 77

also as i can see we are using `develop` branch of typo3 sniffs, we should use stable branch/commit.
so @yastapenko-ScnSoft please update travis file with particular branch for `git clone` command( it has param `-b` to specify branch/commit since git 1.8)

**Pull request closed without merging**